packages/core/fs/src/MemoryFS.js

```diff
@@ -913,7 +913,7 @@ class Directory extends Entry {
   }
 }
 
-function makeShared(contents: Buffer | string): Buffer {
+export function makeShared(contents: Buffer | string): Buffer {
```

Comment:
Are these exports use somewhere?

Reply:
Yes: https://github.com/parcel-bundler/parcel/blob/bcf64468ebe3ada0f913ee7c039f959e35033e06/packages/dev/repl/src/parcel/ExtendedMemoryFS.js#L1

That implements many of the callback and fd based FS operations so that Yarn works